README.md

```diff
@@ -1,33 +1,88 @@
-# Northcoders House of Games API
+# **Northcoders House of Games API**
 
-## Background
+## **Background**
 
 This project is an API with the intention of mimicking a real world backend service. It will be used to provide information to the front end architecture.
 
-The database will be PSQL, and you will interact with it using [node-postgres](https://node-postgres.com/).
+The database uses be PSQL, and is interacted with using [node-postgres](https://node-postgres.com/). It is hosted using [ElephantSQL](https://www.elephantsql.com/)
 
-## How to create the enviroment variables to run locally on your machine
+A version of this API is hosted at
 
-First clone this repo.
+```
+https://nc-games-w79g.onrender.com/
+```
 
-Then to connect to the database's via two .env files. One for the development database and one for the test database.
+Adding a path e.g **/api/reviews** to the end of the url will access different endpoints (all endpoints described in the endpoints section).
 
-Like this
+## **How to Setup**
 
-```
-PGDATABASE=my_database_test
+First clone this repo using the command:
 
-PGDATABASE=my_database
+```
+git clone https://github.com/Nanobot1514/NC-Games-Project.git
 ```
 
-Install all dependencies using
+Then install all dependencies using:
+
+**NOTE: minimum Node version of v19.3.0 and minimum Postgres version of 8.7.3 are required**
 
 ```
 npm install
 ```
 
-Remember to add these files to the .gitignore so you don't the risk of senstitive information about your databases being pushed to github.
+Then to connect to the database's via two .env files (.env.production & .env.test). One for the test database and one for the development database.
+
+Like the following:
+
+```
+PGDATABASE=<my_database_test>
+
+PGDATABASE=<my_database>
+```
+
+If for some reason this process was not followed correctly and the PGDATABASE environment variable was not set correctly, an error would be thrown such as 'PGDATABASE not set'
+
+## **Seed the Databases**
+
+For the development database:
+
+```
+npm run seed
+```
+
+For production:
+
+```
+npm run seed-prod
+```
+
+## **Testing**
+
+All tests can be run using the following command:
+
+```
+npm test app.test.js
```

Comment:
assuming the working directory is correct. as long as they have the repo open the npm test app will run from any directory in there